civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -552,6 +552,10 @@ function civihr_employee_portal_init() {
 function _civihr_employee_portal_set_up_google_analytics_data_layer() {
   global $user;
 
+  if (!function_exists('datalayer_add')) {
```

Comment:
Usually, what we do is to check if the module that declares the function exists, not the function itself. It's very unlikely that another module will have a function with this same name, but not impossible. Please change this to `if (!module_exists('data_layer'))`;